.github/workflows/PR-skip.yml

```diff
@@ -13,6 +13,20 @@ on:
 
 jobs:
   build:
+    # Ensure this matrix matches the one in PR.yml
```

Comment:
I suspect a more useful comment would be "Ensure any changes to this matrix are done in PR-skip.yml" inside PR.yml itself.